### ISSUE TITLE
I CAN NOT get same result

### ISSUE THREAD

**Issue by Mr-zihao** [junfu1115/DANet#41]
i trained the model  using the pretrained model(resnet-101) with cityscapes dataset, but i only got 70.07 mIOU, How can i the same result as the paper? please~~~~  thanks !
I set the batch_size 15 (5 GPUs), epochs 120 .

**Comment by junfu1115** [comment]
@Mr-zihao, I suggest you use our [command](https://github.com/junfu1115/DANet/blob/master/README.md#usage) to train the network, including batch_size 8, epochs 240 etc. In addition, the result of val set is not real during training phase, you need use test.py to obtain the final results of val set

**Comment by Mr-zihao** [comment]
Hello, i still can not get the same result as the paper using the same setting and testing after training. The final result is mIOU 75.05( backbone : resnet101 with multi-grid and DA  without multi-scale).
maybe there are still some tricks. What can i do???   thanks~~~~

**Comment by Mr-zihao** [comment]
@junfu1115 Hello, i still can not get the same result as the paper using the same setting and testing after training. The final result is mIOU 75.05( backbone : resnet101 with multi-grid and DA without multi-scale).
maybe there are still some tricks. What can i do??? thanks~~~~

**Comment by junfu1115** [comment]
@Mr-zihao, There's no other tricks during training except for multi-loss on the end of the network when both PAM and CAM are used. you can ref https://github.com/junfu1115/DANet/issues/14#issue-363473614 although there's still some gaps compared to our results. Or try resnet50 when training the network. We hope you get same results.

**Comment by yazhe2017** [comment]
@junfu1115 Hi, Thanks for the great repo. Why the val mIoU results during training phase is not real? I found the val phase during training is very fast but the 'real' validation from test.py is very slow (it is about take 20min with default setting 4gpus in single scale) ? Thanks.

**Comment by MendelXu** [comment]
Have you guys reproduced d the result? I have trained the model with the same command authors denote and test it on single scale. However, I can only get 78.93 mIOU which is 1 point lower while pixelAcc is 95.99.

**Comment by junfu1115** [comment]
@yazhe2017 In training phase, we evalute the model with only a part of image, it is fast but not real result. while in test phase whole image is used.

**Comment by Serge-weihao** [comment]
@Mr-zihao could you tell me your crop size and total gpu memory use at "batch_size 15 (5 GPUs), epochs 120 " as you mentioned above?
